config/settings/test.py

```diff
@@ -1,4 +1,20 @@
-from .dev import *  # noqa
+from .base import *  # noqa: F401
+
+# SECURITY WARNING: don't run with debug turned on in production!
+DEBUG = True
```

Comment:
According to https://docs.djangoproject.com/en/3.2/topics/testing/overview/#other-test-conditions, this is redundant, so we should probably remove this

Reply:
Previously the code was <from .dev import *  # noqa> hence refactoring to remove DEBUG=true according to the link. Nice one.